aperturedb/cli/ingest.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+import logging
+import typer
+from typing_extensions import Annotated
+from aperturedb.ParallelLoader import ParallelLoader
+from aperturedb.ImageDataCSV import ImageDataCSV
+from aperturedb.BBoxDataCSV import BBoxDataCSV
+from aperturedb.EntityDataCSV import EntityDataCSV
+from aperturedb.BlobDataCSV import BlobDataCSV
+from aperturedb.ConnectionDataCSV import ConnectionDataCSV
+from aperturedb.PolygonDataCSV import PolygonDataCSV
+from aperturedb.VideoDataCSV import VideoDataCSV
+from aperturedb.DescriptorDataCSV import DescriptorDataCSV
+from aperturedb.DescriptorSetDataCSV import DescriptorSetDataCSV
+
+from aperturedb.Utils import create_connector
+from aperturedb.Query import ObjectType
+
+logger = logging.getLogger(__file__)
+app = typer.Typer()
+
```

Comment:
it's nice how the typer framework here allows a nice separation of concern so you know that this command requires those parts of the main interface